### PR TITLE
Example: simple plot, grid data with contours

### PR DESCRIPTION
**Pull request by Rumpkie**

### CODE REVIEW

File: examples/plotting/plot_grid_data_overlay.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+# parameters
+level = 5
+vmin = -8
+vmax = 64
+lat = 36.5
+lon = -98.5
+
+# panel 1, basemap, radar reflectivity and NARR overlay
+ax1 = fig.add_axes(map_panel_axes)
+#display.plot_basemap()
```
Comment (jjhelmus) [inline examples/plotting/plot_grid_data_overlay.py]: This line should be removed.  Dead code should not appear in the examples without an explanation.

File: examples/plotting/plot_grid_data_overlay.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
+"""
+=====================================
```
Comment (jjhelmus) [inline examples/plotting/plot_grid_data_overlay.py]: This line should align with the text below, it is one character too long

File: examples/plotting/plot_grid_data_overlay.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+Create a grid plot with data overlay
+=====================================
```
Comment (jjhelmus) [inline examples/plotting/plot_grid_data_overlay.py]: One character shorter.

File: examples/plotting/plot_grid_data_overlay.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+An example which creates a plot of a gridded NEXRAD radar on a map
+with latitude and NCEP North American regional reanalysis (NARR) pressure
+ is plotted on top of the grid.
```
Comment (jjhelmus) [inline examples/plotting/plot_grid_data_overlay.py]: no space at the beginning of this line, it causes a warning when building the docs.

**Comment by jjhelmus** [comment]
@Rumpkie a few minor changes and this should be good to go.  Let me know if you would prefer if I make these changes.

**Comment by jjhelmus** [comment]
Bump. @Rumpkie are you still working on this PR?

**Comment by scollis** [comment]
Another bump.. Where are we on this?

**Comment by Rumpkie** [comment]
I thought this was done and ready for prime time.

**Comment by scollis** [comment]
Cool.. @jjhelmus will review once he is back

**Comment by Rumpkie** [comment]
If its not I'll make any changes ASAP.

**Comment by jjhelmus** [comment]
@Rumpkie I made a number of suggested changes, can you address these?

**Comment by scollis** [comment]
@Rumpkie : where are we at with this? This needs to be accepted soon

**Comment by scollis** [comment]
@Rumpkie We need to close this out. What is the situation here?

**Comment by Rumpkie** [comment]
FTR: Per our offline discussion, there was some confusion about what was going on with this I am resolving @jjhelmus's suggestions and will update this ASAP.

**Comment by Rumpkie** [comment]
Changes have been commited, this should be good now.

**Pull request merged**